### ISSUE TITLE
UnicodeDecodeError on UTF-8 character.

### ISSUE THREAD

**Issue by kierun** [lgiordani/punch#7]
There is a problem with files that contain UTF-8 character.  Create a file called `version.txt` (which punch has defined in `FILES`) which contains:

```
# ⚠ <- this causes the stack trace.
0.5.1
```

then run `punch --part patch` to get this trace:

```
Traceback (most recent call last):
  File "/home/usr/.virtualenvs/intranet/bin/punch", line 205, in <module>
    updater.update(current_version.as_dict(), new_version.as_dict())
  File "/home/usr/.virtualenvs/intranet/lib/python2.7/site-packages/punch/file_updater.py", line 25, in update
    new_file_content = self.rep.replace(old_file_content, current_version, new_version)
  File "/home/usr/.virtualenvs/intranet/lib/python2.7/site-packages/punch/replacer.py", line 33, in replace
    new_text = new_text.replace(_search_pattern, _replace_pattern)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 2: ordinal
not in range(128)
```

**Comment by lgiordani** [comment]
I can confirm it happens with Python 2.7.10 and not with Python 3.4.3
I will look into this. Thanks

**Comment by kierun** [comment]
My apologies. I should have said it was on Python 2.7.

**Comment by lgiordani** [comment]
Fixed in version 1.1.2. Thank you for finding this issue.

**Comment by kierun** [comment]
You are welcome. ☺
